messaging/methods/media/getMedia.md

```diff
@@ -41,7 +41,7 @@ var a = b;
 {% sample lang="ruby" %}
 
 ```ruby
-# Ruby Example
+downloaded_media = messaging_client.get_media("12345", "mediaId")
```

Comment:
Can you show how to work with the download_media object? 

Saving to disk or pushing to AWS kinda deal

Reply:
Added writing of downloaded media to a file 👍